src/components/Pokemon/Pokemon.js

```diff
@@ -0,0 +1,77 @@
+import React, { useEffect} from 'react';
+import { useSelector, useDispatch } from 'react-redux';
+import PokemonList from './PokemonList/PokemonList';
+import SearchBar from './SearchBar/SearchBar';
+import { fetchDataFromAPI } from '../../store/actions';
+import { Link } from 'react-router-dom';
+import Title from '../Title/Title';
+import styled from 'styled-components';
+import image from '../../backgrounds.jpg';
+const StyledDiv=styled.div`
```

Comment:
Usually, we use styled-components in another file, for example: if you have Pokemon.js component, you want to create another file named Pokemon.style.js, in the same folder, for all the components that use styled-components.